backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -1,14 +1,20 @@
 table {
-  width: 100%;
-  margin-bottom: 15px;
-  border-collapse: separate;
+  // Extend all tables with Bootstrap's default table style
+  @extend .table;
 
   th, td {
-    padding: 7px 5px;
     border-right: 1px solid $color-border;
     border-bottom: 1px solid $color-border;
     vertical-align: middle;
     text-overflow: ellipsis;
+    overflow: hidden;
+
+    // Do not wrap text in all header and content cells.
+    // Overflowing content gets truncated and an ellipsis is shown.
+    // If you want to disable this, add the .wrap-text class to the column.
+    &:not(.wrap-text) {
+      white-space: nowrap;
+    }
```

Comment:
I'm not sure about this. I think our default should be to wrap, and to specify areas where wrapping is undesirable (like actions).